routes/v2/apiApple.js

```diff
@@ -0,0 +1,14 @@
+// eslint-disable-next-line new-cap
+const router = require('express').Router();
+
+router.get('/v2/apple/countries/:country?', async (req, res) => {
+	const { country: countryName } = req.params;
```

Comment:
why would you not just do `const { country } = req.params`

Reply:
the code was already there and it will be deleted when v2 is deprecated anyways